packages/styles/select.css

```diff
@@ -37,51 +72,53 @@
   min-height: var(--select-min-height);
   padding: var(--space-three-quarters) var(--space-smallest);
   padding-right: var(--space-large);
-  border: 1px solid var(--field-border-color);
+  border: 1px solid var(--select-field-border-color);
   font-family: Roboto;
   font-weight: normal;
   font-style: normal;
   font-size: var(--text-size-small);
-  color: var(--field-content-color);
+  color: var(--select-field-text-color);
   text-decoration: rgb(102, 102, 102);
   box-shadow: inset 0 1px 2px rgba(0, 0, 0, 0.05);
   appearance: none;
 }
 
-.Field__select--wrapper select:not([disabled]):hover {
-  border-color: var(--field-border-color-hover);
+.Field__select--wrapper:not(.Field--has-error)
+  select:not([disabled]):not([required]):hover {
+  border-color: var(--select-field-border-color-hover);
 }
 
 .Field__select--wrapper select:focus {
-  border-color: var(--field-border-color-focus);
-  box-shadow: 0 0 0 1px var(--field-border-color-focus),
-    0 0 5px var(--field-border-color-focus-glow),
+  border-color: var(--select-field-border-color-focus);
+  box-shadow: 0 0 0 1px var(--select-field-border-color-focus),
+    0 0 5px var(--select-field-border-color-focus-glow),
     inset 0 1px 2px rgba(0, 0, 0, 0.05);
 }
 
 .Field__select--wrapper select:focus:hover {
-  border-color: var(--field-border-color-focus-hover);
+  border-color: var(--select-field-border-color-focus-hover);
 }
 
 .Field__select--wrapper select:invalid,
 .Field--has-error select {
-  border-width: var(--space-quarter);
-  border-color: var(--field-border-color-error);
+  border-width: 2px;
+  border-color: var(--select-field-border-color-error);
 }
 
 .Field__select--wrapper.Field--has-error select:hover {
-  border-color: var(--field-border-color-error-hover);
+  border-color: var(--select-field-border-color-error-hover);
 }
 
 .Field__select--wrapper select:invalid:focus,
 .Field__select--wrapper.Field--has-error select:focus {
-  border-width: var(--space-quarter);
-  border-color: var(--field-border-color-error);
+  border-width: 2px;
+  border-color: var(--select-field-border-color-error);
   box-shadow: rgba(0, 0, 0, 0.05) 0 1px 2px 0 inset,
-    var(--field-border-color-error-focus-glow) 0 0 4px 0;
+    var(--select-field-border-color-error) 0 0 0 1px,
```

Comment:
This was done after a discussion with @dequelabs/cauldron-design in which we decided a temporary solution to address an a11y issue would be to increase the size of the focus ring upon focus.